File: 01.计算机基础知识/05.设计模式/04.行为型模式/03.责任链模式/java/scut/hujie/chainOfResPattern/ConsoleLogger.java
package scut.hujie.chainOfResPattern;

public class ConsoleLogger extends AbstractLogger {

	public ConsoleLogger(int level) {
		this.level = level;
	}

	@Override
	protected void write(String message) {
		System.out.println("Standard Console::Logger: " + message);
	}

}

File: 01.计算机基础知识/05.设计模式/04.行为型模式/03.责任链模式/c++/ConsoleLogger.cpp
#include "ConsoleLogger.h"


ConsoleLogger::ConsoleLogger()
{
}

ConsoleLogger::ConsoleLogger(int level) :AbstractLogger(level)
{
}

void ConsoleLogger::write(string message) {
	cout << "Standard Console::Logger: " << message << endl;
}

ConsoleLogger::~ConsoleLogger()
{
}
